modules/exploits/osx/local/timemachine_cmd_injection.rb

```diff
@@ -0,0 +1,93 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+  include Msf::Post::File
+  include Msf::Post::OSX::Priv
+  include Msf::Post::OSX::System
+  include Msf::Exploit::EXE
+  include Msf::Exploit::FileDropper
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'          => 'Mac OS X TimeMachine Command Injection Privilege Escalation',
```

Comment:
I'd like to think this won't be the last command injection LPE in TimeMachine. Perhaps mention `tmdiagnose` in the title, or something else unique and more applicable.